diagrams/ecosystemdetail.svg

```diff
@@ -50,7 +50,7 @@
 
 <path class="link" d="m430 175c100 -100 120 -100 125 10"/>
 <text x="550" y="120" role="heading" aria-level="2">Transfer</text>
-<text x="560" y="135">Repeateable</text>
+<text x="560" y="135">Repeatable</text>
```

Comment:
to match the other lowercase instances of `repeatable` in the diagram
```suggestion
<text x="560" y="135">repeatable</text>
```

Reply:
🤦‍♀️